src/domain/event/state/EventReducers.tsx

```diff
@@ -0,0 +1,39 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import { EVENT_ACTIONS } from '../constants/EventActionConstants';
+import { ChildEvent } from '../type/EventChildTypes';
+import { profileQuery_myProfile_children_edges as edges } from '../../api/generatedTypes/profileQuery';
+
+export const defaultChildEventData: ChildEvent[] = [];
+
+export default createReducer(defaultChildEventData, {
+  [EVENT_ACTIONS.CLEAR_EVENT]: (state) => (state = defaultChildEventData),
+  [EVENT_ACTIONS.ENROL_CHILD]: (state, action) => {
+    const item: ChildEvent = {
+      childId: action.payload.childId,
+      eventId: action.payload.eventId,
+    };
+    state.push(item);
```

Comment:
```suggestion
    return [...state, item];
```
So that we get clear return statement.